scripts/socks/README.md

```diff
@@ -31,7 +31,9 @@ $ ./socks.sh     # check status
 $ ./socks.sh off # turn off socks proxy
 ```
 
-The script assumes you have stored your key using "gov" somewhere within name (`~/.ssh/*gov*`). If you put it elsewhere or name it otherwise, you can set the VA_SOCKS_KEYFILE variable before invoking it. The script also sets up a proxy for the "Wi-Fi" network by default; if you are using a different network, you can override the SOCKS_NETWORK environment variable - for example:
+The script assumes you have stored your key at (`~/.ssh/id_rsa_vagov`). If you put it elsewhere or name it otherwise, you can set the VA_SOCKS_KEYFILE variable before invoking it (like this: `VA_SOCKS_KEYFILE=~/.ssh/my_vagov_privatekey`).
+
+The script also sets up a proxy for the "Wi-Fi" network by default; if you are using a different network, you can override the SOCKS_NETWORK environment variable - for example:
```

Comment:
Instead of defaulting to `Wi-Fi`, I think we should default to `networksetup -listallnetworkservices | head -2 | tail -1`. This list is sorted by service order so it gets the primary connection. They could of course still override it using the `SOCKS_NETWORK` env var.

This is probably out of scope of the PR though.

Reply:
when I run that comment I end up with USB 10/100/1000 LAN, so that wouldn't work for my normal day to day. I think 'Wi-Fi" is probably the most common connection modality for people using laptops.

Reply:
`networksetup -listallnetworkservices` returns completely different order for me, too. So I don't think it's a valid option.

Reply:
`networksetup -listnetworkserviceorder` may be what we're looking for. But I'm fine with manually specifying as well.

Reply:
This is what I get:
`networksetup -listnetworkserviceorder
An asterisk (*) denotes that a network service is disabled.
(1) Apple USB Ethernet Adapter
(Hardware Port: Apple USB Ethernet Adapter, Device: en7)

(2) USB 10/100/1000 LAN
(Hardware Port: USB 10/100/1000 LAN, Device: en8)

(3) Wi-Fi
(Hardware Port: Wi-Fi, Device: en0)

(4) Bluetooth PAN
(Hardware Port: Bluetooth PAN, Device: en6)

(5) Thunderbolt Bridge
(Hardware Port: Thunderbolt Bridge, Device: bridge0)`

Reply:
Alright fine:
```
networksetup -listnetworkserviceorder | grep -B1 $(route get example.com | grep interface | awk '{print $2}') | head -1 | cut -f2- -d' '
```

:joy:

It's a little crazy. I'm good with skipping.

Reply:
Let's consider this for another PR. Your approach may work and might make it more bulletproof - but probably needs more testing across different machines.